app/lib/theme/widgets/ContextualHelp.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Tooltip from "@mui/material/Tooltip";
+import { BC_GOV_LINKS_COLOR, TASKLIST_GREY_BG_COLOR } from "../colors";
+
+/**
+ * @param {string} title - The title of the tooltip to be displayed (can be HTML)
```

Comment:
I find "title" misleading because some of our tooltips are going to show sentences and bullets. I know we can't change the MUI prop names, but here we could say "the text the tooltip will display" (or whatever)

Also these params are great documentation